src/modules/ekf2/EKF/ekf.h

```diff
@@ -244,6 +244,8 @@ class Ekf final : public EstimatorInterface
 	Vector3f getAccelBiasVariance() const { return Vector3f{P(13, 13), P(14, 14), P(15, 15)} / sq(_dt_ekf_avg); } // get the accelerometer bias variance in m/s**2
 	Vector3f getMagBiasVariance() const { return Vector3f{P(19, 19), P(20, 20), P(21, 21)}; }
 
+	bool accel_bias_inhibited() const { return _accel_bias_inhibit[0] || _accel_bias_inhibit[1] || _accel_bias_inhibit[2]; }
```

Comment:
there's quite a fight between snakes and camels in that file

Reply:
Yes it's a bit upsetting. Should we kick off 2022 with some bike shedding to finally get it under control? (FYI @MaEtUgR @mcsauder)

My reading of the Google C++ style guide is that simple getters should be named the same as the variable.

I'm flexible though, I just want it to be unambiguous and largely enforceable via tooling.